README.md

```diff
@@ -1,3 +1,34 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+Built with Ruby 2.7.2, CSV, Rspec 3.12.0
+
+
+**We plan to meet daily at 2:30pm MST for stand-ups and work together or be available for most of the afternoon and evening with a dinner break.**
+
+**Our team plans to use _Github Projects_ to organize our tasks/tickets that need to be accomplished in our project. We are also using _Miro_ to organize our thoughts and to mind-map/brainstorm our whole project.  We collectively add inputs when brainstorming and then we can edit tickets in our *Github* projects.** 
+[GitHub Project](https://github.com/users/grace-joh/projects/3)
+[Miro Board](https://miro.com/app/board/uXjVPiopTmI=/)
+
+**To decide on our approach, we looked at a couple ways of organizing our ideas and setting up where we would track everyone's work. We settled on using Miro for planning and brainstorming, and using github projects to organize individual tasks, which we picked over trello because we wanted to try something most of us had not used before.**
+
+## update 
+** **Code design** ^^ **
```

Comment:
I sent you my update in Slack.. sorry I didn't have it ready yesterday!

Reply:
Thanks, and not worries!